packages/cli/src/reporters/util.ts

```diff
@@ -208,3 +208,11 @@ function toString (val: any): string {
     return val.toString()
   }
 }
+
+export function stdOutWrite (text: string) {
+  process.stdout.write(text)
+}
+
+export function stdOutWriteLn (text: string, afterLnCount = 1, beforeLnCount = 0) {
+  process.stdout.write(`${'\n'.repeat(beforeLnCount)}${text}${'\n'.repeat(afterLnCount)}`)
+}
```

Comment:
How else can we call these methods? It feel a bit too verbose to have stdOutWrite in the name. Should we create a class like Writer and have methods like print and println?